src/annotator/config.js

```diff
@@ -10,15 +10,15 @@ var docs = 'https://h.readthedocs.io/en/latest/embedding.html';
  *
  * @param {Window} window_ - The Window object to read config from.
  */
-function config(window_) {
-  var options = {
+function configure(window_) {
```

Comment:
I had a look through the "h" code and indeed, most of the functions that have `configure` prefix do actually modify some aspect of the global environment or a passed in object, so the use of a verb is appropriate there.